src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheckTest.java

```diff
@@ -63,6 +64,7 @@ public void testTokensNotNull() {
     @Test
     public void testWithMultilineStatements() throws Exception {
         final DefaultConfiguration checkConfig = createModuleConfig(OneStatementPerLineCheck.class);
+        checkConfig.addAttribute("treatTryResourcesAsStatement", "true");
```

Comment:
All looks good, but there is nuance.
Such Check did not have any property - https://checkstyle.org/config_coding.html#OneStatementPerLine
and not it has, so we need to follow our rule to make Input to be config specific.

Please update Input files with multiline comment that define configuration of Check it runs and please put trailing comment on place where violation is present.

Reply:
done

Reply:
> please put trailing comment on place where violation is present.

Please do.

Please do comment over class, as not javadoc, content of existing multiline comment, has no value, you can remove it with you comment, and add extra empty lines to keep expected line number the same.

Reply:
done